views/beneficial-owner-individual.html

```diff
@@ -1,8 +1,8 @@
 {% extends "layout.html" %}
 
+{% set title = "Tell us about the individual person" %}
```

Comment:
Don't see a reason to move this out of the block below

Reply:
I think thats why they don't do that in the other files as well